include/triton/Analysis/Allocation.h

```diff
@@ -160,6 +160,15 @@ class Allocation {
             size_t offset = 0)
         : kind(kind), id(nextId++), size(size), alignment(alignment),
           offset(offset) {}
+
+    size_t setOffsetAligned(size_t newOffset) {
```

Comment:
Actually this whole function body can just be replaced by `return offset = llvm::alignTo(newOffset, alignment)`?

Reply:
That's a nice utility. Thanks